freqtrade/constants.py

```diff
@@ -31,7 +31,7 @@
     'properties': {
         'max_open_trades': {'type': 'integer', 'minimum': 0},
         'ticker_interval': {'type': 'string', 'enum': list(TICKER_INTERVAL_MINUTES.keys())},
-        'stake_currency': {'type': 'string', 'enum': ['BTC', 'ETH', 'USDT']},
+        'stake_currency': {'type': 'string', 'enum': ['BTC', 'ETH', 'USDT', 'EUR', 'USD']},
```

Comment:
if we start to support fiat trading - I think we should have a check issuing a warning that fiat_currency should be the same or the conversation will fail (coinmarketcap does not support EUR/USD conversation as far as i know).

it should probably not fail - but warn that conversation fails in this case

Reply:
Good catch, I will see how it behave and update the code